up_bridge/components/graph.py

```diff
@@ -41,74 +41,120 @@ def plan_to_dependency_graph(
 def _partial_order_plan_to_dependency_graph(plan: PartialOrderPlan) -> nx.DiGraph:
     """Convert UP Partial Order Plan to Dependency Graph."""
     dependency_graph = nx.DiGraph()
-    dependency_graph.add_node("end", action="end", parameters=())
+
+    # Prepare Node IDs
+    nodes = set()
```

Comment:
@sebastianstock Unique IDs for partial order plans might still be a problem. Do you think of a scenario to test this case relating to #11 ?

Reply:
To me it looks fine. Why do you think it is a problem for partial order plans?